### PR TITLE
add: text utilities

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Please review my code

### CODE REVIEW

File: components/utilities/text-utilities/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700;800&display=swap");
+* {
+    font-family: "Open Sans", sans-serif;
+}
```
Comment (zoef-yt) [inline components/utilities/text-utilities/text-utilities.css:4]: Why do this here again if mentioned in :root?

**Review by zoef-yt — APPROVED** [review]
Clean code. Looks good to merge.

**Review by developer-anand — APPROVED** [review]
Looks good to merge

File: components/utilities/text-utilities/text-utilities.css (hunk 1 above)
Comment (bhushannaikaroskar) [reply components/utilities/text-utilities/text-utilities.css:4]: things were not getting applied like font-weight. By doing this it was working.

**Pull request merged**